src/current/_includes/releases/v23.2/v23.2.0-alpha.2.md

```diff
@@ -0,0 +1,106 @@
+## v23.2.0-alpha.2
+
+Release Date: October 2, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-0-alpha-2-security-updates">Security updates</h3>
+
+- The `SIGHUP` signal now clears the cached expiration times for [client certificates](https://www.cockroachlabs.com/docs/v23.2/cockroach-cert#how-security-certificates-work) that are reported by the `security.certificate.expiration.client` metric. [#110726][#110726]
+
+<h3 id="v23-2-0-alpha-2-general-changes">General changes</h3>
+
+- Increased the maximum permitted value of the `COCKROACH_RPC_INITIAL_WINDOW_SIZE` environment variable to 64MB. In conjunction with tuning your operating system's maximum TCP window size, this can increase the throughput that Raft replication can sustain over high latency network links. [#111255][#111255]
+
+<h3 id="v23-2-0-alpha-2-sql-language-changes">SQL language changes</h3>
+
+- The `discard` [log message](https://www.cockroachlabs.com/docs/v23.2/logging-overview) is now limited to once per minute by default. The message now includes both the number of transactions and the number of statements that were discarded. [#110805][#110805]
+- The [cluster setting](https://www.cockroachlabs.com/docs/v23.2/cluster-settings) `kv.rangefeed.enabled` no longer controls access to `RANGEFEED SQL` commands. Instead, use `feature.changefeed.enabled`. [#110676][#110676]
+- SQL commands that were previously limited to the `admin` [system privilege](https://www.cockroachlabs.com/docs/v23.2/security-reference/authorization#supported-privileges) can now be used by users with the `VIEWCLUSTERMETADATA` or `REPAIRCLUSTERMETADATA` system privilege, depending on whether the operation is read-only or modifies state. [#110084][#110084]
+- Added a `last_error` column to the `cluster_execution_insights`, `node_execution_insights`, `cluster_txn_execution_insights`, and `node_txn_execution_insights` tables. These columns contain error messages for failed executions. [#110565][#110565]
+- The new backup option `updates_cluster_monitoring_metrics` tracks the timestamp of the last backup failure due to a KMS error. This option is disabled by default. [#104634][#104634]
+- The new restore option `strip_localities` optionally strips the locality information from a backup when restoring to a cluster with different regions than the source cluster.
+
+  Adding a primary region to a regionless restore (with or without regional by row tables) will not work without modifying the database. However, the attempt will produce an error message that provides instructions to alter the restored database.
```

Comment:
> (with or without regional by row tables)

this is will be no longer needed as of #111443, which I plan on merging soon!

> However, the attempt will produce an error message that provides instructions to alter the restored database.

perhaps some version of this might be better?:
"For a cluster restore, this entails dropping the zone config of the database, followed by dropping type d.public.crdb_internal_region. For database/table restores, this entails just dropping type d.public.crdb_internal_region."

Reply:
PTAL at the latest changes and let me know if there is more work for me to do here.

Reply:
LGTM - thank you